Unity-Technologies/ScriptableRenderPipeline
# Disable the new input system for runtime debug menu

### Purpose of this PR
Put the debug menu input code between `ENABLE_LEGACY_INPUT_MANAGER` define.

Added an error message in the debug menu window to warn about debug menu not being available in build when using the new input system
![image](https://user-images.githubusercontent.com/6877923/72614614-02202d80-3933-11ea-9f70-45fa9ec68713.png)

#### QA
There should be no more errors when enabling the new input system package in the player settings and in the build we should not be able to open it since it does not work yet.  
I tested with the input mode both and the debug menu works fine in both build and editor.
